### PR TITLE
Class Settings documentation

### PR DESCRIPTION
**Pull request by udayunnimsft**
Adding documentation for Class Settings to expose settings specific to Teams' of type Class. For now we are exposing a single Boolean value - notifyGuardiansAboutAssignments. This property will only be exposed if the Team type is of type Class.

### CODE REVIEW

File: api-reference/beta/resources/team.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ For more information about working with groups and members in teams, see [Use th
 |memberSettings|[teamMemberSettings](teammembersettings.md) |Settings to configure whether members can perform certain actions, for example, create channels and add bots, in the team.|
 |messagingSettings|[teamMessagingSettings](teammessagingsettings.md) |Settings to configure messaging and mentions in the team.|
 |webUrl|string (readonly) | A hyperlink that will go to the team in the Microsoft Teams client. This is the URL that you get when you right-click a team in the Microsoft Teams client and select **Get link to team**. This URL should be treated as an opaque blob, and not parsed. |
+|teamClassSettings|[teamClassSettings](teamclasssettings.md) |Configure settings specific to team of type Class|
```
Comment (AkJo) [inline api-reference/beta/resources/team.md]: Optional. Configure settings of a class. Available only when the team represents a *Class*.

Comment (AkJo) [reply api-reference/beta/resources/team.md]: Please rephrase as above.

File: api-reference/beta/resources/team.md [hunk 2]
```diff
@@ -95,6 +96,33 @@ The following is a JSON representation of the resource.
 
 ```
 
+If the team is of type Class, there is a classSettings property on the team.
```
Comment (AkJo) [inline api-reference/beta/resources/team.md]: If the team represents a *Class*, there is .....

File: api-reference/beta/resources/teamclasssettings.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+localization_priority: Normal
+author: "nkramer"
+ms.prod: "microsoft-teams"
+---
+
+# teamClassSettings resource type
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Configure settings specific to [team](team.md) of type Class. This is an optional setting, and will only be shown, when the [team](team.md) is of type Class.
```
Comment (AkJo) [inline api-reference/beta/resources/teamclasssettings.md]: Configure *Class*-specific properties of a team. Available only when the team represents a Class.

File: api-reference/beta/resources/team.md [hunk 4]
```diff
@@ -95,6 +96,33 @@ The following is a JSON representation of the resource.
...
+  "internalId": "string",
+  "isArchived": false,
+  "webUrl": "string (URL)",
+  "displayName": "string",
+  "description": "string",
+  "classification": "string",
+  "specialization": "string",
+  "visibility": "string",
+  "classSettings": {"@odata.type": "microsoft.graph.teamClassSettings"}
+}
```
Comment (nkramer) [inline api-reference/beta/resources/team.md]: There should be only one json representation for the type.

File: api-reference/beta/resources/teamclasssettings.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+localization_priority: Normal
+author: "nkramer"
+ms.prod: "microsoft-teams"
+---
+
+# teamClassSettings resource type
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Configure *Class*-specific properties of a team. Available only when the team represents a *Class*.
```
Comment (nkramer) [inline api-reference/beta/resources/teamclasssettings.md]: Add a link back to team.md

**Comment by Lauragra** [comment]
@udayunnimsft , @clearab , this appears to be live in beta, are we ready to merge this PR? Thanks.

**Comment by gagarwal** [comment]
This work is done with https://github.com/microsoftgraph/microsoft-graph-docs/pull/6198.

**Pull request closed without merging**